utils/coreWrapper.ts

```diff
@@ -92,6 +93,11 @@ export class CoreWrapper {
   public async deployMockOrderLibAsync(
     from: Address = this._tokenOwnerAddress
   ): Promise<OrderLibraryMockContract> {
+    const truffleOrderLibrary = await OrderLibrary.new(
```

Comment:
Can we be smarter about this? Ideally this is only deployed once per coreWrapper. Maybe save the instance of OrderLibrary as an instance variable on coreWrapper (like _tokenOwnerAddress)